subsys/dm/dm.c

```diff
@@ -389,14 +394,27 @@ static void dm_thread(void)
 			case TIMESLOT_NORMAL_END:
 				dm_reschedule();
 				if (dm_context.ranging_status) {
+#if (IS_ENABLED(CONFIG_DM_MODULE_RPC_HOST))
```

Comment:
Please avoid using ifdefs if possible:

```suggestion
					   if(IS_ENABLED(CONFIG_DM_MODULE_RPC_HOST)) {
```

Reply:
Thanks for limiting the number of ifdefs! The code looks more elegant now :)